cmds/less/lineio/lineio.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2018 the u-root Authors. All rights reserved
```

Comment:
I'm not familiar with u-root typical package structure, but IMO lineio and sortedmap belong in pkg/... not cmd/less/... They aren't less-specific.